commands/droplets.go

```diff
@@ -385,7 +385,7 @@ func extractUserData(userData, filename string) (string, error) {
 func extractVolumes(volumeList []string) []godo.DropletCreateVolume {
 	var volumes []godo.DropletCreateVolume
 
-	fmt.Printf("extracting volumes from %#v\n", volumeList)
+	fmt.Fprintf(os.Stderr, "extracting volumes from %#v\n", volumeList)
```

Comment:
Would it be possible to create a `notice` message func in errors.go? This way, we don't need to about where text goes.